lib/cf-h1-proxy.c

```diff
@@ -183,7 +183,7 @@ static void h1_tunnel_go_state(struct Curl_cfilter *cf,
     infof(data, "CONNECT phase completed");
     data->state.authproxy.done = TRUE;
     data->state.authproxy.multipass = FALSE;
-    /* FALLTHROUGH */
+    CURL_FALLTHROUGH();
```

Comment:
I think this could drop the `CURL_` prefix since this is an internal macro. To keep it shorter and more similar to what it used to be.

Reply:
My preference goes for a prefix because it says cleary where the symbol is coming from, and also avoids the minor chance of colliding with a symbol coming from a 3rd-party header. We also already use the CURL prefix (or at least a fairly unique name) for most internal symbols. This one is used 85 times in the codebase, so relatively rare.

Are we okay with these?